README.md

```diff
@@ -1,3 +1,7 @@
+# This project is deprecated and is not maintained anymore
+We wrote a [migration guide](https://auth0.com/docs/libraries/lock/v11/migration-lock-passwordless) showing how you can migrate from lock-passwordless to the new [Lock with Passwordless mode](https://github.com/auth0/lock#new-auth0lockpasswordlessclientid-domain-options). If you're encoutering issues, please reach out to our amazing support team at https://support.auth0.com or open [an issue in the new repo](https://github.com/auth0/lock/issues).
```

Comment:
- I'll rephrase the first sentence to something simpler like:
 > If you have used lock-passwordless in the past a migration guide is available [here](link) to implement it using lock v11 (or whatever you want to call it).
- typo: encoutering -> encountering
- I'd remove the "open an issue on LOCK repo". Stick to support.

Things like this should be reviewed by someone else since they look like business decisions, IMO.